app/Http/Controllers/ContactUsController.php

```diff
@@ -2,12 +2,41 @@
 
 namespace App\Http\Controllers;
 
-use Illuminate\Http\Request;
+use App\Http\Requests\ContactUsRequest;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Mail\Message;
 
 class ContactUsController extends Controller
 {
     public function contact()
     {
-        return view('home.contact');
+        return view('contacts.contactUs');
     }
-}
\ No newline at end of file
+
+    public function send(ContactUsRequest $request): RedirectResponse
+    {
+
+        // dd($request->validated());
+        $data = $request->validated();
+        \Log::debug('test', $data);
+
+        \Mail::send(
+            'emails.contactUs',
+            [
+                'firstname' => $data['firstname'],
```

Comment:
At lesson I requested, to have at least one select on the form.

Reply:
looks that select exists but it is not used in mail.

Reply:
added missing elements, but the changes are on  Mailing-dependency-injection branch